### PR TITLE
Set retry times by environment variable

### PR DESCRIPTION
**Pull request by msJinLei**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Accounts/Accounts/Az.Accounts.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '2.2.8'
```
Comment (wyunchi-ms) [inline src/Accounts/Accounts/Az.Accounts.psd1:15]: No need to update psd1 file.

File: src/Accounts/Accounts/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -43,8 +43,8 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 
-[assembly: AssemblyVersion("2.2.8")]
-[assembly: AssemblyFileVersion("2.2.8")]
+[assembly: AssemblyVersion("2.2.77")]
+[assembly: AssemblyFileVersion("2.2.77")]
```
Comment (wyunchi-ms) [inline src/Accounts/Accounts/Properties/AssemblyInfo.cs:47]: No need to update this file.

File: src/Accounts/Authentication/Extensions/ClientFactoryExtension.cs [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        class HttpRetryTimes
+        {
+            static public int? AZURE_PS_HTTP_MAX_RETRIES
+            {
+                get
+                {
+                    return TryGetAzurePsHttpMaxRetries();
+                }
+            }
+            static public int? AZURE_PS_HTTP_MAX_RETRIES_FOR_429
```
Comment (wyunchi-ms) [inline src/Accounts/Authentication/Extensions/ClientFactoryExtension.cs:31]: Is it will be better to rename `429` to `LongRunningOperation`?

**Comment by msJinLei** [comment]
@wyunchi-ms  Sorry the PR is duplicate with https://github.com/Azure/azure-powershell/pull/14950. I am going to close it.

**Pull request closed without merging**